examples/material_gallery/lib/demo/leave_behind_demo.dart

```diff
@@ -67,6 +67,19 @@ class LeaveBehindDemoState extends State<LeaveBehindDemo> {
     }
   }
 
+  void handleUndo(LeaveBehindItem item) {
+    int insertionIndex = leaveBehindItems.length;
```

Comment:
in "real" apps, this list can easily be thousands if not hundreds of thousands of items long. Is there any way we can avoid walking it?